swim/node.go

```diff
@@ -58,7 +58,9 @@ type Options struct {
 func defaultOptions() *Options {
 	opts := &Options{
 		StateTimeouts: StateTimeouts{
-			Suspect: 5000 * time.Millisecond,
+			Suspect:   5 * time.Second,
+			Faulty:    1 * time.Hour,
```

Comment:
The one hour default I put in here was just a wild guess. I don't see why it is beneficial to keep faulty nodes in the list for that long at the moment since faulty nodes are not used by anything yet.

When we decide we want to keep them longer we can change it while adding a feature that benefits from keeping the faulties around. I'm also happy with bumping the faulty time to a longer period if there is a good argument to keep them around for a longer period.